### ISSUE TITLE
Should we put aria-hidden="true" on the background behind Modal Dialogue

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/aria-practices#238]
VoiceOver and TalkBack may end up outside the modal dialogue as user swipes through the dialogue, if the background is not set with aria-hidden, regardless of trapping the tab key etc. to keep focus in the modal. Suggest aria-hidden on background div.

    <div class="full-content-wrapper" aria-hidden="true">
        ... full content of page
       </div>
    <div role="dialogue" class="modal" ... >...</div>

Unless there is a better solution for mobile screen readers.

**Comment by mcking65** [comment]
I do not think it is good to use aria-hidden in this way. If the background were visually obscured and completely inert, then it would be appropriate. But, if users can click outside the dialog to close it, or if users can see anything outside the modal, then it should be up to the screen reader to decide how to represent the modal experience.

We will soon have dialog examples that can be used as references for test cases. And, at that point, they could be used to raise bugs against TalkBack or VoiceOver if they do not render the experience in a way that is consistent with what is experienced by people who do not use a screen reader.

My $0.02 ... thoughts?

**Comment by DavidMacDonald** [comment]
I'm not sure... I don't have enough user data. I know that @IanPouncey of the Pacillo group recommends aria-hidden on the background.

**Comment by mcking65** [comment]
This issue was resolved via discussion and commits referenced in issue #325.
